tests/e2e/e2e_groups_test.go

```diff
@@ -162,16 +159,18 @@ func (s *IntegrationTestSuite) prepareGroupFiles(c *chain, adminAddr string, mem
 			Metadata: "Bob",
 		},
 	}
+
 	s.writeGroupMembers(c, members, originalMembersFilename)
 
-	newMembers := append(members, GroupMember{
+	newMembers := append(members, group.MemberRequest{
 		Address:  member3Address,
 		Weight:   "1",
 		Metadata: "Charlie",
```

Comment:
Maybe use different meta data from charlieAddr below

Reply:
Metadata field in this case is irrelevant as this is test data, changing it has no effect on the test, and in this particular case, we're targeting the same member in both so it makes sense to keep it consistent. However within the scope of this function we should actually be using `member3Address` passed in and not the globally available `charlieAddr`

Reply:
I know metadata is not relevant to the test outcome, but I still think its good to keep both test cases clean and not imply connections between, its visually distracting.

Reply:
Right but in this case the metadata is denoting the specific user and is meant to be the same entity...the only thing that's supposed to change is the weight